package.json

```diff
@@ -29,6 +30,31 @@
     "lib",
     "build/lib"
   ],
+  "scripts": {
```

Comment:
linking @KazuCocoa : this should probably be done for all other dependencies, so we could eventually completely get rid of gulp usage